.github/workflows/release-go-bindings.yaml

```
@@ -0,0 +1,174 @@
+# This is the monolithic release flow described in ADR 25
+name: Release Go Bindings
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
```

actionlint (1.7.12)

[error] "options" can not be set to "bump" input because its input type is not "choice"

(events)